tests/Kernel/Patterns/OrganisationTeaserPatternRenderingTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Kernel\Patterns;
+
+use Drupal\Tests\oe_theme\Kernel\AbstractKernelTestBase;
+
+/**
+ * Test Organisation teaser pattern rendering.
+ */
+class OrganisationTeaserPatternRenderingTest extends AbstractKernelTestBase {
+
+  /**
+   * Test that Organisation teaser pattern is correctly rendered.
+   *
+   * @param array $fields
+   *   Fields data.
+   * @param array $assertions
+   *   Test assertions.
+   *
+   * @throws \Exception
+   *
+   * @dataProvider dataProvider
+   */
+  public function testOrganisationTeaserPatternRendering(array $fields, array $assertions) {
+    $pattern = [
+      '#type' => 'pattern',
+      '#id' => 'organisation_teaser',
+      '#fields' => $fields,
+    ];
+
+    $html = $this->renderRoot($pattern);
+    $this->assertRendering($html, $assertions);
+  }
+
+  /**
+   * Data provider for testOrganisationTeaserPatternRendering.
+   *
+   * @return array
+   *   An array of test data arrays with assertations.
```

Comment:
Typo.

Reply:
fixed. also it was fixed in  tests/Kernel/Patterns/FilePatternRenderingTest.php